roles/network_plugin/calico/templates/calico-node.yml.j2

```diff
@@ -239,6 +238,10 @@ spec:
             - name: IP
               value: "autodetect"
 {% endif %}
+{% if enable_dual_stack_networks %}
+            - name: IP6
+              value: autodetect
```

Comment:
Do we need IP6_AUTODETECTION_METHOD (see 5 lines before)

Reply:
What would you want to be set to? [Here are the options](https://docs.projectcalico.org/reference/node/configuration#ip-autodetection-methods). It's presently using the default of `first-found`

Reply:
`first-found` is far from ideal as soon as you have multiple interfaces and want to use CrossSubnet options
see https://github.com/kubernetes-sigs/kubespray/commit/7d7739e0311b8290aa257015c75a278caf1b2052
Is there the IPv6 equivalent of `status.hostIP`

Reply:
Not that I could see...

Maybe adding a variable like `calico_ip6_can_reach_address:` with a default value like `google.com`? That way it can be overridden for the user's scenario

Reply:
I'm wondering, do we really need both IP and IP6 ? if you run dual-stack, making tunnels over IPv4 makes more sense (less overhead), so is IP6 used for dual stack ?
On bare metal I usually have the K8S traffic on an isolated VLAN
This is fine with me for a first version, I'll think more about it when I try dual stack in the future.

Reply:
A case that caused issues for me, when I tried to just have pods and services dual-stacked but the hosts not, was the pods making outbound connections to services. They detect IPv6 being present, so resolve AAAA addresses, then try and route to them using IPv6, which then can't leave the SDN as there's no IPv6 routing in the physical network.

Another scenario might be Calico advertising the IPv6 pod / service subnets via BGP. That would give routing directly to pods and services from outside SDN where you wouldn't want to use an ingress. Without dual-stacked hosts, the traffic couldn't arrive.

Reply:
I was not thinking of having IPv4 only host, but thinking about Calico not needing to know anyhting about the actual IPv6 config of the host. Thanks for the BGP case, in this case Calico indeed the IPv6 info.
Has you have a dual stack cluster handy, do we have any fields with the host IPv6 in the pods status ?

Reply:
The only IPv6 IPs seem to be in `podIPs`
```yaml
status:
  conditions:
  - lastProbeTime: null
    lastTransitionTime: "2020-12-14T10:31:57Z"
    status: "True"
    type: Initialized
  - lastProbeTime: null
    lastTransitionTime: "2020-12-14T10:31:59Z"
    status: "True"
    type: Ready
  - lastProbeTime: null
    lastTransitionTime: "2020-12-14T10:31:59Z"
    status: "True"
    type: ContainersReady
  - lastProbeTime: null
    lastTransitionTime: "2020-12-14T10:31:57Z"
    status: "True"
    type: PodScheduled
  containerStatuses:
  - containerID: docker://5a7c2d65e4d1b56a0668dbf16daee4126b00c7bbd2cc757859afe388af0652ab
    image: myregistry.com/tenant/app:image-v0.1.0
    imageID: docker-pullable://myregistry.com/tenant/app@sha256:ec6cffc50892bcf8eaa5957c7adb4040b0f9977a41a93c2a8c384ef94b45ebbf
    lastState: {}
    name: app
    ready: true
    restartCount: 0
    started: true
    state:
      running:
        startedAt: "2020-12-14T10:31:58Z"
  hostIP: 172.28.239.105
  phase: Running
  podIP: 10.233.104.85
  podIPs:
  - ip: 10.233.104.85
  - ip: fd52:5b5b:b0ab:f430::1:bf
  qosClass: BestEffort
  startTime: "2020-12-14T10:31:57Z"
```

Reply:
Shall we get this out of the door, and then make some iterative changes to it? It's functional and fits most use cases as it ism and it would be good to get something into master for people to tweak and improve upon 🙂

Reply:
yes, maybe just wait for 2.15 release (soon) then we can merge

Reply:
Looks like 2.15 went on on Thursday. Did this get forgotten about?